config/event_schemas/SequenceRunStateChange.json

```diff
@@ -11,14 +11,9 @@
         "type": "object",
         "required": [
           "detail-type",
-          "resources",
           "detail",
-          "id",
```

Comment:
I wonder if we should keep this.... 
It may be useful for deduplication or specific events in the history/logs where the other fields are not descrimitive enough?

Reply:
Ok, I will add `id` back.

Reply:
Fixed with https://github.com/umccr/orcabus/commit/af95b34f10cfad73874dacff9579da6bccd63120